Formula/a/azion.rb

```diff
@@ -24,6 +24,7 @@ def install
       -X github.com/aziontech/azion-cli/pkg/constants.StorageApiURL=https://api.azion.com
       -X github.com/aziontech/azion-cli/pkg/constants.AuthURL=https://sso.azion.com/api
       -X github.com/aziontech/azion-cli/pkg/constants.ApiURL=https://api.azionapi.net
+      -X github.com/aziontech/azion-cli/pkg/metric.SegmentKey=Irg63QfdvWpoANAVeCBEwfxXBKvoSSzt
```

Comment:
What is this magic key? And why is it needed?

Reply:
Now that you mentioned it, we do not really need to inject this here 🤔 We were hyperfocusing on another issue. 
I'll remove this, but keep the test change (behavior changed internally)